shared/src/fixtures/tables/user_session_logs.rs

```diff
@@ -7,17 +7,21 @@ use time::Date;
 #[derive(Debug, PartialEq, FromRow, StructOfArray, Serialize, Deserialize)]
 pub struct UserSessionLogsTable {
     pub id: i32,
-    pub event_date: Date,
-    pub user_id: i32,
-    pub event_name: String,
-    pub session_duration: i32,
-    pub page_views: i32,
-    pub revenue: BigDecimal,
+    pub event_date: Option<Date>,
+    pub user_id: Option<i32>,
+    pub event_name: Option<String>,
+    pub session_duration: Option<i32>,
+    pub page_views: Option<i32>,
+    pub revenue: Option<BigDecimal>,
```

Comment:
Should note that this makes the table a little more cumbersome to use, as you always have to `Some(...)` around values so they work in `assert_eq!`. 

Not a big deal, but thought I'd point it out.

Reply:
Since the columns are nullable, `Option` is the more appropriate type. This made it easier for me to check that inserting null types works.